github/libp2p.yml

```
@@ -3181,6 +3181,11 @@ repositories:
     files:
       .github/workflows/stale.yml:
         content: .github/workflows/stale.yml
+    teams:
+      admin:
```

Do we need anything beyond `write` or `maintain`? https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization
```suggestion
      maintain:
```
or
```suggestion
      push:
```

---

Let's go with that first then and expand if we need it :)